science/lenstool/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+name                lenstool
+version             7.1.1
+categories          science
+license             GPL
+platforms           darwin
+maintainers         lam.fr:eric.jullo openmaintainer
```

Comment:
please add your github id here. i.e.

maintainers {lam.fr:eric.jullo @ejullo} openmaintainer

Reply:
Thanks for your quick reply. I've done the change in the Portfile

Reply:
Thanks, but you also need to squash your commits, so there is only one in this PR, that follows correctly follows the commit guidelines.